docs/api/field_type_reference.md

```diff
@@ -1881,15 +1881,19 @@ $validators = [
 
 ## RichText Field Type
 
+!!! caution "Deprecated"
+
+    The RichText Field Type is deprecated since v2.4. Relying on any class from the `eZ\Publish\Core\FieldType` namespace is deprecated.
+
+    If you're implementing any interface or extending any base class from that namespace, refer to its PHPDoc to see what to implement or extend instead.
```

Comment:
This sentence is ok, should be connected to the message above.

Reply:
It is, it's in the same caution message, but it is more readable/has more impact with a small space above.